src/App.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import "./Apptesting.css";
+import Todo from "./compo/Todo";
```

Comment:
Please use PascalCase for File and Folder names and specify complete name "Components"